cli/main.go

```diff
@@ -25,6 +27,7 @@ var (
 	defaultListenAddr         = getEnv("BOOST_LISTEN_ADDR", "localhost:18550")
 	defaultRelayCheck         = os.Getenv("RELAY_STARTUP_CHECK") != ""
 	defaultGenesisForkVersion = getEnv("GENESIS_FORK_VERSION", "")
+	defaultRelayMinBidEth     = getEnvFloat64("MIN_BID_ETH", 0.001)
```

Comment:
One more thing, the default minimum bid should be zero. It's up to the user (operator) to set this value based on their preferences. I'm just concerned with the side effects of this. For example, a default value of `0.001` would affect testnets (like [sepolia](https://builder-relay-sepolia.flashbots.net)) where the average mev reward is most likely to be below that threshold. I understand the desire for a non-zero default: low value blocks will be built using the local execution engine. But when CL clients start to compare the value to the block it built locally, I think that would serve as a better "default" minimum.

Reply:
Sure, that makes sense.